packages/console/src/pages/Connectors/components/Guide/index.tsx

```
@@ -31,10 +32,10 @@ const Guide = ({ connector, onClose }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { id: connectorId, type: connectorType, name, configTemplate, readme } = connector;
 
-  const locale = i18next.language;
+  const localeRaw = i18next.language;
+  const locale = languageEnumGuard.parse(localeRaw);
```

will fallback to?

---

we set 'en' as fallbackLng when initializing i18next, so I think i18next.language will be 'en' if not specified?

---

`parse` will throw an error?

---

I thought that if `localRaw` is not in the `name`'s keys, it is informative enough to throw ZodError to show that `localRaw` is currently not supported?
Or should we throw an error in some other error types?

---

I think this should fallback to `en`, instead of killing the app.

---

Besides, this is a refactor PR, is not expected to change the logic.

---

sounds reasonable, will fix this soon.

---

Updated, PTAL